classes/class-wc-retargeting-tracking.php

```diff
@@ -80,16 +84,28 @@ function init_form_fields()
         $this->form_fields = array(
             'domain_api_key' => array(
                 'title' => __('Tracking API KEY'),
-                'description' => __('Insert Retargeting TRACKING API Key. <a href="https://retargeting.biz/admin?action=api_redirect&token=5ac66ac466f3e1ec5e6fe5a040356997" target="_blank" rel="noopener noreferrer">Click here</a> to get your Domain API Key'),
+                'description' => __('Insert Retargeting TRACKING API Key. <a href="https://retargeting.biz/admin/module/settings/docs-and-api" target="_blank" rel="noopener noreferrer">Click here</a> to get your Tracking API Key'),
```

Comment:
Am renuntat la el ca ma trimite pe https://retargeting.biz/admin/module/discounts care da 404 - not found.